.zenodo.json

```diff
@@ -0,0 +1,37 @@
+{
+	"title": "My Research Software",
```

Comment:
Is this intentionally not `REPO_NAME` (also below in lines 13: `FULLNAME`, line 10: `SHORT_DESCRIPTION`, lines 20/25 `ZENODO_RECORD` etc.) – this should maybe be added to the to do list in `README.md` more detailed so you do not miss anything small like the

Reply:
You are right, we should fix that. Probably also in `CITATION.CFF`

Reply:
We might want to remove `CITATION.CFF` and `.zenodo.json` example files all together and improve the `README.md`. Simplify stuff?

Reply:
I agree. For `CITATION.cff` we could add an item to the to do list in `README.md`: "Go to https://citation-file-format.github.io/cff-initializer-javascript/#/ to create a CITATION.cff file and add it to the repository's root directory".